src/node_api.cc

```diff
@@ -123,6 +187,22 @@ v8::Local<v8::Value> V8LocalValueFromJsValue(napi_value v) {
   return local;
 }
 
+napi_status V8NameFromPropertyDescriptor(napi_env env,
```

Comment:
@boingoing Having an anon namespace would be nice because none of the symbols for the utilities here need to be exported, like `static` would indicate too. You could keep the `v8impl`/`uvimpl` namespaces and just wrap their entire content in a anon namespace if you like that for readability.